tutorials/javaInstallation.md

```diff
@@ -8,4 +8,4 @@
 
 * [**:fab-apple: Mac**](tutorials/javaInstallationMac.html) -- If you are on a Mac and you anticipate using JavaFX in your projects, you are using the precise JDK specified in this guide.
 * [**:fab-windows: Windows**](tutorials/javaInstallationWindows.html)
-* %%**:fab-linux: Linux**: coming soon ...%%
+* [**:fab-linux: Linux**](tutorials/javaInstallationLinux.html)
```

Comment:
Related to #96, this should not include the prefix `tutorials/` as it is already included in the [Java Installation Guides URL](https://se-education.org/guides/tutorials/javaInstallation.html) (I believe this change is not yet in `master` however).

Reply:
@mfjkri Thanks for pointing this out! I'll modify this accordingly for line 11 only (I think lines 9 and 10 should be updated when I sync from master).